crates/viewer/re_view_spatial/src/ui_3d.rs

```diff
@@ -102,9 +102,7 @@ impl View3DState {
         scene_bbox: &SceneBoundingBoxes,
         scene_view_coordinates: Option<ViewCoordinates>,
     ) {
-        // Mark as interaction since we want to stop doing any automatic interpolations,
-        // even if this is caused by a full reset.
-        self.last_eye_interaction = Some(Instant::now());
```

Comment:
Did you mean to do that? Seems out of place.

Reply:
thanks. That's from a different PR and slipped in